applicationinsights-react-js/package.json

```diff
@@ -72,7 +72,7 @@
     },
     "peerDependencies": {
         "tslib": "*",
-        "react": "^17.0.1",
+        "react": ">= 17.0.1",
```

Comment:
From some of the issues, I thought that there was some incompatibility between 17 and 18? Can you please validate before we just open the range